### ISSUE TITLE
Failed open on macOS Big Sur

### ISSUE THREAD

**Issue by CherryLover** [Jermic/Android-Crack-Tool#31]
![image](https://user-images.githubusercontent.com/18376501/112951390-e0911780-916d-11eb-88b0-00725f73b3cc.png)

after` sudo spctl --master-disable` have same issue

**Comment by gitgeek4dx** [comment]
M1芯片？需要恢复模式启动后，修改系统安全策略为‘宽容安全性’，可以通过菜单栏的安全设置修改，也可以通过终端输入：csrutil disable，然后正常启动进入系统后执行csrutil status确认已经关闭，再次执行 sudo spctl --master-disable

**Comment by CherryLover** [comment]
不是 M1 的，是 Inter 的版本，我自己电脑是从上一个版本升级到 Big Sur 的就始终无法使用，公司的电脑，安装的系统就是 Big Sur ，就没有任何问题，我晚点时候按照你的方法尝试下。

**Comment by chiuan** [comment]
> 

m1 可以运行的嘛
